README.md

```diff
@@ -1,3 +1,5 @@
+<div align="center"><img src="https://raw.githubusercontent.com/PlasmaPy/PlasmaPy-logo/master/exports/with-text-dark.png" width="600"/></div>
```

Comment:
If I am reading this correctly, this sets a fixed width for the logo.  I'm wondering how this will appear on mobile devices if the image width is greater than the screen width.  This isn't a big deal and is easy to change, so I'm happy to approve this now.

Reply:
![screenshot_20190123-200118_firefox](https://user-images.githubusercontent.com/11289391/51630189-cc2bb180-1f49-11e9-84a4-f862b375004e.jpg) I think it's gonna work all right :)